metabase/metabase
# Handle metric filters in drilldowns

Fixes https://github.com/metabase/metabase/issues/6010
Reproduces https://github.com/metabase/metabase/issues/6010

How to test:
- Go to Admin -> Data Model -> Metrics and create a metric with Filter (Total > 150) and Aggregation (Count)
- New -> Question -> Orders -> Visualize
- Summarize -> Remove the default aggregation -> Add metric -> Choose your metric
- Group by Created At (month)
- Click on any dot -> View these Orders
- Make sure the newly constructed query has the filter from the metric applied

Metric definition:
<img width="1728" alt="Screenshot 2022-09-23 at 23 04 11" src="https://user-images.githubusercontent.com/8542534/192048308-523341cb-0364-427a-a352-935a40c9f104.png">

Adding the metric to a question:
<img width="445" alt="Screenshot 2022-09-23 at 23 05 07" src="https://user-images.githubusercontent.com/8542534/192048409-55ca2264-c528-4966-8a88-7011830e78c8.png">

Drilldown:
<img width="791" alt="Screenshot 2022-09-23 at 23 04 01" src="https://user-images.githubusercontent.com/8542534/192048338-ef59257c-b3e1-4f19-8914-c67154157985.png">

New query with the metric filter:
<img width="1272" alt="Screenshot 2022-09-23 at 23 03 43" src="https://user-images.githubusercontent.com/8542534/192048367-3b8b8d32-edad-486e-af41-594baf9de018.png">
